src/lib/request/requestManager.ts

```diff
@@ -38,7 +38,18 @@ interface RequestsManagerParams {
 function getRESTAPI(endpoint: string): string {
   const apiData = new URL(endpoint);
   // e.g 'https://api.snyk.io/rest/'
-  return new URL(`${apiData.protocol}//api.${apiData.host}/rest`).toString();
+
+  let newUrl = new URL(
+    `${apiData.protocol}//api.${apiData.host}/rest`,
+  ).toString();
+
+  if (endpoint.includes('app.')) {
```

Comment:
this can be simplified and always replace `app.` if found directly above 

like so `  const apiData = new URL(endpoint.replace("app.", ''))`